src/Elliptic/BoundaryConditions/ApplyBoundaryCondition.hpp

```diff
@@ -74,8 +74,11 @@ void apply_boundary_condition(
           elliptic::BoundaryConditions::BoundaryCondition<Dim>,
           BoundaryConditionClasses, DbTagsList>::type;
   call_with_dynamic_type<void, boundary_condition_classes>(
-      &boundary_condition, [&map_keys_to_direction, &box,
-                            &fields_and_fluxes...](const auto* const derived) {
+      &boundary_condition,
+      []<typename... LocalFieldsAndFluxes>(
+          const auto* const derived, const MapKeys& local_map_keys_to_direction,
+          const db::DataBox<DbTagsList>& local_box,
+          const LocalFieldsAndFluxes&... local_fields_and_fluxes) {
```

Comment:
Is it better to take as `&&`?

Reply:
If I remember correctly, I tried and NVCC refused to compile that. I already spent a pretty large amount of time getting this code to compile at all because of the nested lambdas. Please feel free to revisit, but I'd like to keep it as-is. I also expect the arguments to be `const&` not rvalue, but I don't know the elliptic code well enough.